### ISSUE TITLE
--onlyunknown crashes

### ISSUE THREAD

**Issue by SimenB** [davglass/license-checker#49]
```
scanning /home/testyo/repos/frontend-repo/code
/home/testyo/npm/lib/node_modules/license-checker/lib/index.js:180
                    if (data[item].licenses.indexOf('*') > -1 ||
                                           ^
TypeError: Cannot read property 'indexOf' of null
    at /home/testyo/npm/lib/node_modules/license-checker/lib/index.js:180:44
    at Array.forEach (native)
    at /home/testyo/npm/lib/node_modules/license-checker/lib/index.js:164:34
    at /home/testyo/npm/lib/node_modules/license-checker/node_modules/read-installed/read-installed.js:138:5
    at /home/testyo/npm/lib/node_modules/license-checker/node_modules/read-installed/read-installed.js:251:14
    at cb (/home/testyo/npm/lib/node_modules/license-checker/node_modules/read-installed/node_modules/slide/lib/async-map.js:47:24)
    at /home/testyo/npm/lib/node_modules/license-checker/node_modules/read-installed/read-installed.js:251:14
    at cb (/home/testyo/npm/lib/node_modules/license-checker/node_modules/read-installed/node_modules/slide/lib/async-map.js:47:24)
    at /home/testyo/npm/lib/node_modules/license-checker/node_modules/read-installed/read-installed.js:251:14
    at cb (/home/testyo/npm/lib/node_modules/license-checker/node_modules/read-installed/node_modules/slide/lib/async-map.js:47:24)
```

`npm@2.11.3`

```
"devDependencies": {
  "archiver": "^0.14.4",
  "autoprefixer": "^6.0.3",
  "babel-core": "^5.8.20",
  "babel-eslint": "^4.1.2",
  "babel-loader": "^5.3.2",
  "babel-runtime": "^5.8.20",
  "browser-sync-webpack-plugin": "^1.0.0",
  "css-loader": "^0.19.0",
  "deepmerge": "^0.2.10",
  "del": "^2.0.2",
  "download": "^4.2.0",
  "eslint": "^1.6.0",
  "eslint-config-airbnb": "^0.1.0",
  "extract-text-webpack-plugin": "^0.8.2",
  "faux-jax": "^4.0.0",
  "file-loader": "^0.8.4",
  "gulp": "^3.9.0",
  "gulp-eslint": "^1.0.0",
  "gulp-jscs": "^3.0.1",
  "gulp-stylint": "^3.0.0",
  "gulp-task-listing": "^1.0.1",
  "gulp-util": "^3.0.6",
  "handlebars-loader": "^1.1.0",
  "html-webpack-plugin-extra-files": "^1.4.0",
  "iectrl": "^0.1.1",
  "imports-loader": "^0.6.4",
  "isparta": "^3.1.0",
  "isparta-loader": "^1.0.0",
  "jasmine-core": "^2.3.4",
  "jasmine-jquery": "^2.1.1",
  "jscs": "^2.2.1",
  "json-loader": "^0.5.2",
  "karma": "^0.13.6",
  "karma-chrome-launcher": "^0.2.0",
  "karma-coverage": "^0.5.0",
  "karma-jasmine": "^0.3.6",
  "karma-junit-reporter": "0.3.3",
  "karma-phantomjs-launcher": "^0.2.0",
  "karma-sourcemap-loader": "^0.3.5",
  "karma-webpack": "^1.7.0",
  "lodash.clonedeep": "^3.0.2",
  "lolex": "^1.2.2",
  "merge-stream": "^0.1.8",
  "node-libs-browser": "^0.5.2",
  "node-noop": "0.0.1",
  "phantomjs": "^1.9.17",
  "phantomjs-polyfill": "0.0.1",
  "postcss-loader": "^0.6.0",
  "require-dir": "^0.3.0",
  "rfile": "^1.0.0",
  "run-sequence": "^1.1.2",
  "style-loader": "^0.12.4",
  "stylint-stylish": "^1.1.2",
  "stylus-loader": "^1.3.0",
  "url-loader": "^0.5.6",
  "webpack": "^1.10.5",
  "webpack-config": "^0.8.0",
  "webpack-dev-server": "^1.11.0"
},
"dependencies": {
  "backbone": "1.1.2",
  "backbone-deep-model": "^0.10.4",
  "backbone-validation": "0.11.5",
  "backbone.layoutmanager": "0.9.6",
  "backbone.modelbinder": "1.1.0",
  "block-ui": "2.70.1",
  "bluebird": "^2.10.1",
  "body-parser": "^1.13.3",
  "bowser": "1.0.0",
  "client-sessions": "^0.7.0",
  "cookie-parser": "^1.3.5",
  "core-js": "^1.2.0",
  "crc": "^3.3.0",
  "datatables": "1.10.5",
  "deep-diff": "0.3.2",
  "express": "^4.13.3",
  "express-session": "^1.11.3",
  "handlebars": "1.3.0",
  "jq-pagination": "1.4.1",
  "jquery": "^1.11.3",
  "jquery.inputmask": "3.1.63",
  "moment": "2.9.0",
  "nanoscroller": "0.8.7",
  "node-uuid": "^1.4.3",
  "string.prototype.startswith": "^0.2.0",
  "underscore": "1.8.3",
  "winston": "^1.0.1"
}
```

**Comment by paazmaya** [comment]
I am facing the same issue

**Comment by davglass** [comment]
Sorry for the delay, I didn't see this issue. Push a fix in `license-checker@4.2.0`

**Comment by paazmaya** [comment]
Thank you mr @davglass ! Here, have some healthy :grapes:
